sections/content_plots/case_evolution.R

```diff
@@ -1,5 +1,18 @@
+output$selectize_evolByCanton <- renderUI({
+  selectizeInput(
+    "selectize_evolByCanton",
+    label    = "Select Kanton",
+    choices  = c("Alle", unique(data_evolution$name))[order(unique(data_evolution$name))],
```

Comment:
Ordering seems to be messed up. This should solve it
```suggestion
    choices  = c("Alle", unique(data_evolution$name)[order(unique(data_evolution$name))]),
```

Reply:
cheers, i was already googling and complaining about ordering with Shiny :)